airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/spec.json

```diff
@@ -4,20 +4,82 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Zendesk Chat Spec",
     "type": "object",
-    "required": ["start_date", "access_token"],
-    "additionalProperties": false,
+    "required": ["start_date", "credentials"],
```

Comment:
remove credential from required because we need to support old spec files as well.